texteditor/buffer.go

```diff
@@ -84,13 +84,12 @@ type Buffer struct { //types:add
 	// autoSaving is used in atomically safe way to protect autosaving
 	autoSaving bool
 
-	// Changed indicates if the text has been Changed (edited) relative to the
-	// original, since last EditDone
-	Changed bool
-
-	// NotSaved indicates if the text has been changed (edited) relative to the
-	// original, since last Save
-	NotSaved bool
+	// notSaved indicates if the text has been changed (edited) relative to the
```

Comment:
Why not just export NotSaved?

Reply:
see note elsewhere -- we might need a mutex to protect that field -- this makes that possible without api changes.